### ISSUE TITLE
NullPointerException when playing a newly loaded score in the REPL while the old score is still playing

### ISSUE THREAD

**Issue by elyisgreat** [alda-lang/alda#203]
I tried to play a score loaded into the REPL, which normally works fine, and I got this:

```
msv> :play

msv> java.lang.NullPointerException
	at clojure.lang.RT.intCast(RT.java:1177)
	at alda.sound.midi$stop_note_BANG_.invokeStatic(midi.clj:76)
	at alda.sound$eval6998$fn__6999.invoke(sound.clj:140)
	at clojure.lang.MultiFn.invoke(MultiFn.java:233)
	at alda.sound$play_BANG_$fn__7051$fn__7054.invoke(sound.clj:217)
	at alda.sound.proxy$java.lang.Object$ScheduledCommand$22a8b854.run(Unknown Source)
	at com.jsyn.engine.SynthesisEngine.processScheduledCommands(SynthesisEngine.java:461)
	at com.jsyn.engine.SynthesisEngine.generateNextBuffer(SynthesisEngine.java:427)
	at com.jsyn.engine.SynthesisEngine.run(SynthesisEngine.java:373)
	at java.lang.Thread.run(Thread.java:745)
Mar 09, 2016 9:36:55 PM com.jsyn.engine.SynthesisEngine run
INFO: JSyn synthesis thread in finally code.

```
When this happened, the sound played forever on a sustained note. I had to quit not only the REPL but close the entire terminal window.

What is going on?

**Comment by daveyarwood** [comment]
Hmm... that's new to me. Do you have the score you were trying to play handy?

Also, did you try to play the score while something else was still playing back? I've noticed buggy behavior around that before.

**Comment by elyisgreat** [comment]
yes. However it is still modified. I'll give you the latest version.
I'm not sure.
Here is the full terminal output.
[Terminal Saved Output.txt](https://github.com/alda-lang/alda/files/166554/Terminal.Saved.Output.txt)

**Comment by daveyarwood** [comment]
I've seen a similar stacktrace before when trying to play a score before the one currently playing is done playing. In my case, it wasn't in the REPL, but while playing score files directly from the command line.

I think the problem has to do with the general way an Alda server handles scores, which is something I'm improving currently via issue #153. It's a little hard to explain, but basically, when you load an Alda score currently, it will affect the way that server handles subsequent file loads, which is not good. We want to essentially "start fresh" when you load a new score. The improvements I'm making should fix this.

I will keep this issue open for now. Once I've made the changes I'm working on, we can revisit this and see if we can reproduce the problem.

**Comment by elyisgreat** [comment]
Ok. I think I can reproduce the issue now. It happens when you *load **and** play* a score into the REPL while a score is still playing. I could send you the score but I think it happens with any score.

here, try it with [this one](https://github.com/alda-lang/alda/files/170731/test.alda.zip)

**Comment by daveyarwood** [comment]
This issue is fixed in 1.0.0-rc15.
